pywhat/identifier.py

```diff
@@ -36,7 +56,9 @@ def identify(self, text: str, dist: Distribution = None,
             # a file in hex format
             identify_obj["File Signatures"] = self._file_sig.check_magic_nums(text)
 
-        identify_obj["Regexes"] = self._regex_id.check(text, dist)
+        identify_obj["Regexes"] = sorted(
+            self._regex_id.check(text, dist), key=key, reverse=reverse
+        )

```

Comment:
Does this run everytime. even when no `key` is provided and it defaults? I was actively trying to take advantage of the fact that the json file is sorted and you don't need to sort after-the-fact as this saves a lot of time when repeatedly calling the API 😄

Reply:
Yes, it does. I may add keys.none key that would not do any sorting.

Reply:
Please! 😄  Just a small scale thing

Reply:
Hey, not sure how:
>  Keys.NONE disables sorting.
Disables sorting if it still runs `sorted()`? 

Other than that, everything looks fine! :D